src/app/features/videomania/index.js

```
@@ -70,13 +106,23 @@ const videoManiaSlice = createSlice({
         state.error = action?.payload;
       })
       .addCase(getVideosByCategory.pending, (state) => {
-        state.isFetching = true;
+        state.isVideoFetching = true;
       })
       .addCase(getVideosByCategory.fulfilled, (state, action) => {
```

ESLint

[error] 'action' is defined but never used.

(no-unused-vars)